src/common.cpp

```diff
@@ -232,20 +231,20 @@ namespace irods::http
         log::debug("{}: Bearer token: [{}]", __func__, bearer_token);
 
         // Verify the bearer token is known to the server. If not, return an error.
-        const auto* object = irods::process_stash::find(bearer_token);
-        if (!object) {
-            log::error("{}: Could not find bearer token matching [{}].", __func__, bearer_token);
+        const auto object{irods::http::process_stash::find(bearer_token)};
```

Comment:
Can we name this something to make it a little more indicative of what it is?

Reply:
Should be a bit better now?